src/lib/fragments/AgGrid.react.js

```diff
@@ -65,6 +65,9 @@ const RESIZE_DEBOUNCE_MS = 200;
 // Rate-limit for updating columnState when interacting with the grid
 const COL_RESIZE_DEBOUNCE_MS = 500;
 
+// Time between syncing cell value changes with Dash
+const CELL_VALUE_CHANGED_DEBOUNCE_MS = 1;
```

Comment:
0 might even be sufficient here, these events are likely coming in from sync code, if so all you need to do is wait for the next tick in the event loop. But that's just FYI, let's leave it as 1, there could be other problems with 0.

Reply:
I think you are right! I played around with different values, but i didn't seem to matter if it was 0, 1 or 100.